app/views/hq/orphans/show.html.haml

```diff
@@ -0,0 +1,127 @@
+%div.action_items
+  %div.instance
+    = link_to 'Edit Orphan', edit_hq_orphan_path(@orphan.id),
+      class: 'btn btn-default', role: 'button'
+    = link_to 'Link to Orphan', '#',
```

Comment:
In the ActiveAdmin interface, creation of sponsorships takes place only from a sponsor's show view. Let's keep it the same in HQ for now, and make the change if the client requests it.